src/esw/brushed_motors/ROSHandler.h

```diff
@@ -19,24 +20,53 @@ class ROSHandler {
     // This holds the ROS Node.
     inline static ros::NodeHandle* n;
 
-    // This keeps track of all the open loop subscribers and publishers
+    // RA
     inline static std::vector<std::string> RANames;
```

Comment:
I think it'd be best practice to use std::array for fixed size things like this

Reply:
ok i changed it on my end